### PR TITLE
Add support for css prop on pre-built GlamorousComponents

### PR DESCRIPTION
**Pull request by tkh44**
**What**:
Add support for `css` prop on pre-built GlamorousComponents

**Why**:
1. It allows the use of computed properties without using `{...{}}` in jsx. 
2. You can write inline styles as objects.

```jsx
const { Div } = glamorous;

export default ({ theme }) => (
  <Div
    css={{
      cursor: 'pointer',
      fill: theme.color.blue[0],
      height: 27,
      transition: 'all 200ms ease',
      width: 90,
      [theme.breakpoints.mq[1]]: {
        height: 27,
        width: 90,
      },
      [theme.breakpoints.mq[2]]: {
        height: 40,
        width: 134,
      },
      [theme.breakpoints.mq[3]]: {
        height: 50,
        width: 165,
      },
    }}
  >
    <Logo />
  </Div>
);
```

**How**:
I just removed the barrier to attach cssOverrides to pre-built GlamorousComponents

Docs needed, but this should be a good start.

#74

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -198,6 +198,13 @@
       "contributions": [
         "example"
       ]
+    },
+    {
+      "login": "tkh44",
+      "name": "Kye Hohenberger",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/662750?v=3",
+      "profile": "https://github.com/tkh44",
+      "contributions": []
```
Comment (kentcdodds) [inline .all-contributorsrc]: Want to add code, test, and doc to this? Then run `npm start contrib.gen`

**Comment by tkh44** [comment]
Since it is implied throughout that the `css` prop is for overriding styles I didn't think much more needed to be added to the docs. I did add an example of using the `css` prop though.

**Comment by tkh44** [comment]
This is really fun to use!
```jsx
<Ul css={{ listStyle: 'none', margin: 0, padding: 0 }}>
  {rooms.map(room => (
    <Li
      css={{
        '& + &': {
          marginTop: 5,
        },
        fontSize: 18,
        height: 40,
        lineHeight: '20px',
      }}
    >
      <Span
        css={{
          '& > svg': {
            fill: theme.color.gray[3],
            height: 14,
            width: 14,
          },
          marginRight: 10,
        }}
      >
        <Plus />
      </Span>
      {room.name}
    </Li>
  ))}
</Ul>
```

**Comment by kentcdodds** [comment]
Not sure what's up with travis... 🤔

**Comment by kentcdodds** [comment]
Well, it passed before, so I'll go ahead and merge this.

**Pull request merged**